packages/xstate-wallet/src/messaging.ts

```diff
@@ -103,7 +110,14 @@ export class MessagingService implements MessagingServiceInterface {
   public async sendChannelNotification(
     method: ChannelClosingNotification['method'] | ChannelUpdatedNotification['method'],
     notificationData: ChannelResult
-  ) {
+  );
+  // eslint-disable-next-line no-dupe-class-members
+  public async sendChannelNotification(
```

Comment:
Can this be removed?

Reply:
The overload?

Reply:
The overload is intentional:
- if you want to respond to `ChannelProposed`, you _must_ provide the `fundingStrategy`
- otherwise, you _must not_ provide the `fundingStrategy`